addon/helpers/ember-sparkles/arc.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import { arc } from 'd3-shape';
+
+export function emberSparklesArc([ radius ], textLabel=false) {
+  if (textLabel === true){
+    return arc().outerRadius(radius - 40).innerRadius(radius - 40);
```

Comment:
this means we will be able to remove the `if (textLabel...)` part (since the arguments will be different depending on whether these are text labels or not)